libs/sdk-ui-dashboard/api/sdk-ui-dashboard.api.md

```diff
@@ -3769,6 +3796,9 @@ export const selectObjectAvailabilityConfig: OutputSelector<DashboardState, Obje
 // @alpha
 export const selectPermissions: OutputSelector<DashboardState, IWorkspacePermissions, (res: PermissionsState) => IWorkspacePermissions>;
 
+// @internal
+export const selectPersistedDashboard: OutputSelector<DashboardState, IDashboard<IDashboardWidget> | undefined, (res: DashboardMetaState) => IDashboard<IDashboardWidget> | undefined>;
+
```

Comment:
why this export?

Reply:
We need get sharedObject: IAccessControlAware & IAuditableUsers;